calico-cloud/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -72,8 +74,23 @@ You must download and set the executable flag each time you get a new version of
 ./tigera-scanner version
 {{imageassuranceversion}}
    ```
+### Integrate the scanner into your build pipeline
+
+You can include the CLI scanner in your CI/CD pipelines (for example, Jenkins, Github actions). Ensure the following:
+
+- The CLI binary is available in the worker or runner where the pipeline is executing
+- If you are running an ephemeral environment in the pipeline, include the download, and update the executable steps in your pipeline to download the scanner on every execution
+- Create a secret containing the API-Token and make it available in the pipeline (for example, using a SECURE_API_TOKEN environment variable)
```

Comment:
We should also mention needing to set the API URL here as well. The scanner always needs both the URL and the auth token.

Reply:
Added API-URL